code/modules/hydroponics/seeds.dm

```diff
@@ -638,13 +638,13 @@
 	if(new_trait?.can_add(src))
 		genes += new_trait.Copy()
 
-	// Adjust stats based on graft stats.
-	src.lifespan	= round(clamp(max(src.lifespan,		(src.lifespan	+(2/3)*(snip.lifespan	-src.lifespan)		)),0,100))
-	src.endurance	= round(clamp(max(src.endurance,	(src.endurance	+(2/3)*(snip.endurance	-src.endurance)		)),0,100))
-	src.production	= round(clamp(max(src.production,	(src.production	+(2/3)*(snip.production	-src.production)	)),0,100))
-	src.weed_rate	= round(clamp(max(src.weed_rate,	(src.weed_rate	+(2/3)*(snip.weed_rate	-src.weed_rate)		)),0,100))
-	src.weed_chance	= round(clamp(max(src.weed_chance,	(src.weed_chance+(2/3)*(snip.weed_chance-src.weed_chance)	)),0,100))
-	src.yield		= round(clamp(max(src.yield,		(src.yield		+(2/3)*(snip.yield		-src.yield)			)),0,10	))
+	// Adjust stats based on graft stats
+	set_lifespan(round(max(lifespan,		(lifespan	+ (2/3)*(snip.lifespan - lifespan)			))))
+	set_endurance(round(max(endurance,		(endurance	+ (2/3)*(snip.endurance - endurance)		))))
+	set_production(round(max(production,	(production	+ (2/3)*(snip.production - production)		))))
+	set_weed_rate(round(max(weed_rate,		(weed_rate	+ (2/3)*(snip.weed_rate - weed_rate)		))))
+	set_weed_chance(round(max(weed_chance,	(weed_chance+ (2/3)*(snip.weed_chance - weed_chance)	))))
+	set_yield(round(max(yield,				(yield		+ (2/3)*(snip.yield - yield)				))))
```

Comment:
Remove the tab indentation, it's hideous.

Reply:
agreed